blog/2021-09-24-yellow-defi-is-now-an-official-sponsor-of-supabase.mdx

```diff
@@ -8,13 +8,20 @@ author_image_url: https://hackernoon.com/_next/image?url=https%3A%2F%2Fcdn.hacke
 tags: [baas-services, firebase, supabase, yellow-defi, yellow-incubator]
 ---
 
+import supabaseLogoUrl from './img/supabase_logo.png';
```

Comment:
store img in static/img

try to put img like this, if it works, change extension back to .md
`
![Architecture](/img/components/architecture.png)`